operator/node.go

```diff
@@ -157,6 +149,14 @@ func (n *operatorNode) Start(logger *zap.Logger) error {
 		logger.Fatal("duty scheduler exited with error", zap.Error(err))
```

Comment:
don't we need to move this code block (with wait and errcheck) after all the code below?
Wait basically blocks forever.
Now start will block until after the initial fetch which is what we wanted.